keyboards/pizzakeyboards/slice65/info.json

```diff
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "Pizzakeyboards",
+    "keyboard_name": "Slice65",
+    "maintainer": "mm0nte",
+    "board": "GENERIC_STM32_F303XC",
+    "bootloader": "stm32-dfu",
+    "debounce": 10,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "nkro": true
```

Comment:
```suggestion
        "nkro": true,
        "mousekey": true,
        "extrakey": true
```

Reply:
Oh I see, these features are added to info.json.